lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -16,21 +16,24 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const tmpl = this.dom.cloneTemplate('#tmpl-lh-perf-metric', this.templateContext);
     const element = this.dom.find('.lh-perf-metric', tmpl);
     element.id = audit.result.name;
+    // FIXME(paulirish): currently this sets a 'lh-perf-metric--undefined' class on error'd audits
     element.classList.add(`lh-perf-metric--${Util.calculateRating(audit.result.score)}`);
```

Comment:
just add a conditional on `const rating = Util.calculateRating(audit.result.score); if (rating !== undefined) {element.classList.add(...);}`? Since `lh-perf-metric--error` is added below that seems like it's fine (unless you have more specific future plans)

Reply:
Right now scores are 0 in the case of error's. :( 
I split that into a second PR because it opens a big can of worms.